repository.datadog.yaml

```
@@ -2,3 +2,4 @@
 schema-version: v1
 kind: stale-branches
 max_age: 4380h # 6 months
+gitlab_check_enable: false
```

I think you need to add 
```suggestion
---
schema-version: v1
kind: mergequeue
enable: true
gitlab_check_enable: false
```

because you're configuring the wrong king here

---

You are right, committed your suggestion 👍